src/global/list.rs

```diff
@@ -282,11 +283,16 @@ pub async fn list_all_global_environments(
     if message.is_empty() {
         println!("No global environments found.");
     } else {
-        println!(
-            "Global environments as specified in '{}'\n{}",
-            console::style(project.manifest.path.display()).bold(),
-            message
+        let header = format!(
+            "Global environments as specified in '{}'",
+            project.manifest.path.display()
         );
+        if show_header {
+            println!("{}", header);
+        } else {
+            tracing::info!("{}", header);
```

Comment:
I think we can hide it in tracing::debug - if it's not necessary information for the user to be shown in stdout, then let's make it the same as `tracing::debug!("Loaded config from: {}", path.display());` above

Reply:
Removed it!